test/Microsoft.Data.Sqlite.Tests/SqliteParameterTest.cs

```diff
@@ -165,13 +166,18 @@ public void Bind_is_noop_on_unknown_parameter()
         [InlineData(double.PositiveInfinity, double.PositiveInfinity)]
         [InlineData(float.NegativeInfinity, double.NegativeInfinity)]
         [InlineData(float.PositiveInfinity, double.PositiveInfinity)]
-        public void Bind_works(object value, object coercedValue)
+        public void Bind_works(object value, object coercedValue, SqliteType? sqliteType = null)
         {
             using (var connection = new SqliteConnection("Data Source=:memory:"))
             {
                 var command = connection.CreateCommand();
                 command.CommandText = "SELECT @Parameter;";
-                command.Parameters.AddWithValue("@Parameter", value);
+                var sqliteParameter = command.Parameters.AddWithValue("@Parameter", value);
+                if (sqliteType.HasValue)
+                {
+                    sqliteParameter.SqliteType = sqliteType.Value;
```

Comment:
does need check here. null will anyway fall back to default

Reply:
sqliteParameter.SqliteType is not nullable and sqliteType.Value throws if sqliteType has no value.